models/action.go

```diff
@@ -562,6 +560,7 @@ func changeIssueStatus(repo *Repository, doer *User, ref string, refMarked map[i
 
 // UpdateIssuesCommit checks if issues are manipulated by commit message.
 func UpdateIssuesCommit(doer *User, repo *Repository, commits []*PushCommit, branchName string) error {
+	initKeywordsRegexp()
```

Comment:
Why not parse when loading settings?

Reply:
Because sadly I cannot reference `code.gitea.io/gitea/models` from `code.gitea.io/gitea/modules/setting` (circular reference)